source/api_documentation/calls_in_progress_api/_update_custom_data.rst

```diff
@@ -42,8 +42,8 @@
         "call_start_time_utc": "2022-12-10T13:45:00.000Z",
         "transfer_start_time_utc": "2022-12-10T13:49:00.000Z",
         "call_status": "bridged",
-        "custom_data": [
-            {"source": "CallsInProgressAPI", "name": "Call Disposition", "value": "CUSTOMER_HANGUP", "data_type": "Short Text"}
-        ]
+        "custom_data": {
```

Comment:
maybe we should add a second key and value here just to show it can have multiple